TSPL.docc/ReferenceManual/Attributes.md

```diff
@@ -345,6 +345,68 @@ struct MyStruct {
   ```
 -->
 
+### backDeployed
+
+Apply this attribute to a function, method, subscript, or computed property
+to include a copy of the symbol's implementation
+in programs that call or access the symbol.
+You use this attribute to annotate symbols that ship as part of an SDK
```

Comment:
Is there any way to make this statement even more firm?   Really clarify that this attribute is not meant for apps to implement on their own, just for platform providers.  Thanks :)

Reply:
@tshortli Thoughts?  I think this should already be clear that `@backDeployed` isn't for apps — an app doesn't "ship part of an SDK".  But there might be a stronger version of this we should use.

Reply:
There are 3rd party SDKs out there (e.g. https://github.com/facebook/facebook-ios-sdk) which this attribute still wouldn't be useful for, so I don't know if the word "SDK" is quite specific enough. Maybe "operating system runtime" would be a reasonable replacement for SDK?

Reply:
Thanks.  Changed in 61e657ab to use "platform" and list code APIs that ship with an OS as an example.

I had originally written SDKs because the SE proposal frames this is terms of SDKs and "resilient libraries".

Reply:
Yes, much better, thank you.